### PR TITLE
Use wakatime-cli standalone

### PR DESCRIPTION
**Pull request by gandarez**

### CODE REVIEW

File: WakaTime.sublime-settings [hunk 1]
```diff
@@ -3,10 +3,6 @@
 // This settings file will be overwritten when upgrading.
 
 {
-    // Your api key from https://wakatime.com/#apikey
```
Comment (alanhamlett) [inline WakaTime.sublime-settings]: We need these default settings, since some people configure their api key in Sublime without wakatime.cfg.

Comment (gandarez) [reply WakaTime.sublime-settings]: added back

File: WakaTime.sublime-settings [hunk 2]
```diff
@@ -30,8 +30,5 @@
     "status_bar_coding_activity": true,
 
     // Obfuscate file paths when sending to API. Your dashboard will no longer display coding activity per file.
-    "hidefilenames": false,
```
Comment (alanhamlett) [inline WakaTime.sublime-settings]: We also still need these two configs.

File: WakaTime.py [hunk 3]
```diff
@@ -384,7 +384,7 @@ def process_queue(timestamp):
 
     # Prevent sending heartbeats more often than SEND_BUFFER_SECONDS
     now = int(time.time())
-    if timestamp != LAST_HEARTBEAT['time'] and LAST_HEARTBEAT_SENT_AT > now - SEND_BUFFER_SECONDS:
+    if isCliInstalled() and timestamp != LAST_HEARTBEAT['time'] and LAST_HEARTBEAT_SENT_AT > now - SEND_BUFFER_SECONDS:
```
Comment (alanhamlett) [inline WakaTime.py]: No, that needs to be an `or`. I'll make the fix on `master` branch.

Comment (gandarez) [reply WakaTime.py]: tks

**Pull request merged**